packages/js/src/ai-assessment-fixes/components/ai-assessment-fixes-button.js

```diff
@@ -85,26 +106,28 @@ const AIAssessmentFixesButton = ( { id, isPremium } ) => {
 
 	// Add tooltip classes on mouse enter and remove them on mouse leave.
 	const handleMouseEnter = useCallback( () => {
-		// Add tooltip classes on mouse enter
-		setButtonClass( "yoast-tooltip yoast-tooltip-w" );
-	}, [] );
+		const direction = isEnabled ? "yoast-tooltip-w" : "yoast-tooltip-nw";
+		setTooltipClass( `yoast-tooltip ${ direction }` );
+	}, [ isEnabled ] );
 
 	const handleMouseLeave = useCallback( () => {
-		// Remove tooltip classes on mouse leave
-		setButtonClass( "" );
+		setTooltipClass( "" );
 	}, [] );
 
 	return (
-		<>
+		<div
+			aria-label={ ariaLabel }
+			className={ tooltipClass }
+			onMouseEnter={ handleMouseEnter }
+			onMouseLeave={ handleMouseLeave }
+		>
```

Comment:
In my opinion we don't need this div. and to refactor the setButtonClass() function. We can handle the `tooltip` with the [onPointerEnter](https://react.dev/reference/react-dom/components/common#handling-pointer-events) and onPointerLeave events from react. We might just need to add a CSS property to the disabled pseudo class in the CSS. 

```
.ai-button:disabled {
	pointer-events: auto;
}
``` 

https://github.com/Yoast/wordpress-seo/assets/152272313/e6079a0c-31db-46f4-a516-e196bd400a13


And then we can toggle the `ariaLabel` of the tooltip according to the disabled state.

Reply:
I haven't gotten this to work, so I'll happily have you take over from here 👍 In the technical choices above I had described why I went with this solution. I hadn't looked into `onPointerEnter` and apparently it works on your side so feel free to refactor it.

Reply:
Nicer solution indeed, thanks a lot! 🙏